### ISSUE TITLE
BUG   Testimonials section is hide the bubble 

### ISSUE THREAD

**Issue by AftabMankapure** [apu52/Travel_Website#294]
**Describe the bug**

Testimonials section is hide the bubble 
bubble is not display on the section

**Expected behavior**

Bubble should be display above the section 

**Screenshots (Mandatory)**

### Current 
![testOld](https://github.com/apu52/Travel_Website/assets/125949765/97a7da5d-267c-4794-8a89-ff15d84d7a9b)

### New

![TestiNew](https://github.com/apu52/Travel_Website/assets/125949765/835bc1f3-7a7f-45eb-9579-d99c1befdf9a)

**Additional Context (Mandatory )**

***Are you contributing under any open-source program ?***

JWOC

**Comment by AftabMankapure** [comment]
@apu52  pls assign me under JWOC

**Comment by apu52** [comment]
@AftabMankapure I will assign you this only after completion of the previous PR

**Comment by apu52** [comment]
@AftabMankapure You are assigned. please make sure to submit your PR before 10 PM tomorrow

**Comment by mandrita16** [comment]
Please assign me under JWOC'24

**Comment by apu52** [comment]
> Please assign me under JWOC'24

This issue is already closed . try to raise any new issue you want to work on
